### PR TITLE
Correct use of render function for Symfony greater than 2.6

### PR DESCRIPTION
**Pull request by dajuam**
Fixes #182

I made a minor change in render function to be compatible with Symfony 3.1

### CODE REVIEW

File: Resources/views/Form/form_admin_fields.html.twig [hunk 1]
```diff
@@ -51,12 +51,12 @@ file that was distributed with this source code.
         <span id="field_actions_{{ id }}" class="field-actions">
             <span id="field_widget_{{ id }}" class="field-short-description">
                 {% if sonata_admin.field_description.associationadmin.id(sonata_admin.value) %}
-                    {% render url('sonata_admin_short_object_information', {
+                    {{ render(url('sonata_admin_short_object_information'), {
```
Comment (greg0ire) [inline Resources/views/Form/form_admin_fields.html.twig]: Why are you also closing the parenthesis for the call to `url` here?

Comment (dajuam) [inline Resources/views/Form/form_admin_fields.html.twig]: Yes, my bad. I tested that way and worked. I'm going to change it now.

**Comment by greg0ire** [comment]
LGTM. Either squash your commits as described in CONTRIBUTING.md or let us do it for you.

**Comment by Soullivaneuh** [comment]
Also please fill the PR template before merge.

**Comment by greg0ire** [comment]
ping @dajuam

**Comment by greg0ire** [comment]
Closed in favor of #188

**Pull request closed without merging**